### ISSUE TITLE
"Failure during lint analysis" with @Composable-annotated modifier extension function

### ISSUE THREAD

**Issue by GSala** [slackhq/compose-lints#315]
Artifact used:
```
com.slack.lint.compose:compose-lint-checks:1.3.1
```

No `lint.xml`

Example code:
```kotlin
@Composable
private fun Modifier.mediumPadding() = this.padding(16.dp)
```

Result:
```
Lint found 1 errors, 15 warnings. First failure:
  
/.../MainActivity.kt: Error: Unexpected failure during lint analysis of MainActivity.kt (this is a bug in lint or one of the libraries it depends on)
```
---

The modifier function should not be annotated with `@Composable` but it was in a function in a project where we were introducing the lint checks for the first time. It failed the lint instead of flagging it as an issue, or ignoring it.

**Comment by ZacSweers** [comment]
Please paste the associated stacktrace that is printed after, the bit you pasted isn't enough info :)

**Comment by GSala** [comment]
Can't reproduce in 1.4.1
